test/test.ts

```diff
@@ -613,6 +613,13 @@ test('is.number', t => {
 	testType(t, 'number', ['integer', 'safeInteger', 'infinite']);
 });
 
+test('is.propertyKey', t => {
+	// ??? Unsure what is the correct syntax to `testType` multiple types. Also are the exclusion relevant?
```

Comment:
I recommend looking at the `testType` source and see if you can figure it out yourself first. I don't remember exactly how it works either.

Reply:
As far as understand it, the test works and is 'good enough'